### ISSUE TITLE
Write "scrappy research" approach

### ISSUE THREAD

**Issue by aassoiants** [aassoiants/bitcoin-ux-research-toolkit#1]
**Link to the google doc**
[here](https://docs.google.com/document/d/1NecZTjz8Nozoqn5eaY1xt83tql9G78Qw_KTdn3GuI1g/edit)

**Format**
1. Why to use the method
- This could include best use cases.
2. How to use the method

**Who's working on this right now?**
@aassoiants 

**Can I contribute and how?**
[Write up template](https://docs.google.com/document/d/1ju6EKEKExKOnUxhv9scAEIlE35I4R4OU3Jq-LSaDuWg/edit#)

**Comment by GBKS** [comment]
Not sure if it's accurate, but I think the title refers to "just doing it" and not turning every bit of research into a PhD thesis. There's a book called "Just enough research" ([PDF](https://www.xinsighters.com/wp-content/uploads/2022/07/9-just-enough-research.pdf)), I really like the title of that, as well as the spirit in which it was written:

> There are dozens of books about applied qualitative research and related techniques out there. The good ones are many hundreds of pages long. Most were written by professional researchers for professional researchers. Very thorough individuals, professional researchers. Most of them are quite charming at parties.
> You, however, are not a professional researcher, which means you need a book written for you—a book that covers a lot of useful ground in few words and makes some of the basic concepts and techniques more accessible.

Research can be really simple, just asking people a few questions, or showing them something and seeing how they respond. I think that in open-source, we are pretty close to some parts of our audience through forums, chats, Twitter, etc, which is great. But there are gaps in which some structured research is needed. Better to do that in a scrappy way than not at all.

**Comment by aassoiants** [comment]
@GBKS that's exactly where this comes from. 

And when we ran our survey, we heard folks being like "i don't have time or resources to run your fancy research, give me something practical." This is for them.

**Comment by GBKS** [comment]
FWIW, it's the same in other parts of design. Some people like reading the Bitcoin Design Guide (RTFM), lots of people just want to look at the screen mock-ups (the "solution", in a way).

I think there's also an element of starting simple and scrappy and then leveling up over time as people/projects get comfortable with the process. But you have to start simple, otherwise they never get to the "later levels". Like level 1 in a lot of video games is a basic orientation, and then you get better and better until you face the boss in level 100 and can only defeat it by perfectly executing a specific 100-button combo in 0.1 seconds while doing a handstand blindfolded.

**Comment by aassoiants** [comment]
@GBKS @mouxdesign good for review: https://docs.google.com/document/d/1NecZTjz8Nozoqn5eaY1xt83tql9G78Qw_KTdn3GuI1g/edit

**Comment by GBKS** [comment]
@aassoiants awesome. Looks great, just had a few comments you can ignore or implement, as you desire. Is it possible to add bitcoin-related references? That's the one thing I found missing.

**Comment by aassoiants** [comment]
Thanks @GBKS , great edits!

Will work with @mouxdesign to bitcoinify the article

**Comment by mouxdesign** [comment]
Looks pretty good to me as well @aassoiants. Thank you for putting this together. I've added in bitcoin related references at the bottom of 3 things that we did withing the community which I think might fall into this scrappy research approach.

One  more suggestion would be to see if we can make any of the text into a visual, one or 2 visuals would add a nice balance. Will see what I can come up with.
